src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -62,5 +73,37 @@ public void setAllowCreatingCaseBeforePaymentsAreProcessed(boolean allowCreating
         this.allowCreatingCaseBeforePaymentsAreProcessed = allowCreatingCaseBeforePaymentsAreProcessed;
     }
 
+    public String getSurnameOcrFieldName(String formType) {
+        return formTypeToSurnameOcrFieldMappings.get(formType);
+    }
+
+    public void setFormTypeToSurnameOcrFieldMappings(List<FormFieldMapping> formTypeToSurnameOcrFieldMappings) {
+        this.formTypeToSurnameOcrFieldMappings = formTypeToSurnameOcrFieldMappings.stream()
+            .collect(
+                toMap(
+                    FormFieldMapping::getFormType,
+                    FormFieldMapping::getOcrField,
+                    (v1, v2) -> {
+                        throw new InvalidConfigurationException(
+                            String.format("Form type has multiple mappings to surname fields %s, %s.", v1, v2)
+                        );
+                    }
+                )
+            );
+//                HashMap::new,
+//                (map, mapping) -> {
+//                    String formType = mapping.getFormType();
+//                    if (map.containsKey(formType)) {
+//                        String msg = String.format("Form type %s has multiple mappings to surname fields.", formType);
+//                        log.error(msg);
+//                        throw new InvalidConfigurationException(msg);
+//                    } else {
+//                        map.put(formType, mapping.getOcrField());
+//                    }
+//                },
+//                HashMap::putAll
+//            );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Comment has incorrect indentation level 0, expected is 8, indentation should be the same level as line 81.](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4498197)